package.json

```diff
@@ -27,10 +27,12 @@
     "eslint": "^7.18.0",
     "husky": "^5.1.0",
     "jest": "^26.6.3",
+    "jest-circus": "^26.6.3",
```

Comment:
I dont see where this is being used. Should be configured in the `testRunner` option of Jest's config if I am not mistaken

Reply:
You're right. The fix was in already existing dependencies so I only updated eslint and the eslint typescript plugin.